client/proposalmsgs_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	"perun.network/go-perun/channel/test"
 	"perun.network/go-perun/client"
 	clienttest "perun.network/go-perun/client/test"
+	_ "perun.network/go-perun/wire/perunio/serializer" // wire serialzer init
```

Comment:
Could we move this file to `wire/perunio/test` since it seems to be specific to perunio?